build/index.md

```diff
@@ -28,56 +30,74 @@ and much improved functionality that can be powerful tools for improving your
 builds' performance or reusability of your Dockerfiles, and it also introduces
 support for complex scenarios.
 
-## Docker Build features
+Docker Build is way more than the `docker build` command and is not only about
+packaging your code, it's a whole ecosystem of tools and features that support
+you not only with common workflow tasks but also provides you with support for
+more complex and advanced scenarios:
+
```

Comment:
Maybe out of scope for this PR, but I think we need more details on what BuildKit is, since the rest of our docs refer to Buildx/BuildKit freely, so we should introduce these concepts as early as possible.

At the very least, we need to clarify:
- The client (build/buildx) - server (buildkit) architecture
- `docker build` vs `docker buildx build`
- Legacy vs BuildKit

I don't think we need much space for this, but we should go over it.

Reply:
Yes in `Docker Build > Overview` that sgtm